vars/uploadRPM.groovy

```diff
@@ -35,6 +36,11 @@ def call() {
                         `find . -name '*.el8.noarch.rpm' -o -name '*.el8.x86_64.rpm'` \
                         ${USER}@repo.ci.percona.com:\${path_to_build}/binary/redhat/8/x86_64/
                 fi
+                if [ `find . -name '*.el9.noarch.rpm' -o -name '*.el9.x86_64.rpm' | wc -l` -gt 0 ]; then
+                    scp -o StrictHostKeyChecking=no -i ${KEY_PATH} \
+                        `find . -name '*.el9.noarch.rpm' -o -name '*.el9.x86_64.rpm'` \
+                        ${USER}@repo.ci.percona.com:\${path_to_build}/binary/redhat/9/x86_64/
+                fi
```

Comment:
we are adding the same block the fourth time. the only difference is the os number.
So I suggest adding the loop and reusing code instead of just adding new blocks

Reply:
Good point. Fixed!